bin/webdriver-manager

```diff
@@ -260,12 +261,19 @@ for (name in binaries) {
 
 switch (argv._[0]) {
   case 'start':
-    if (!binaries.standalone.exists) {
+    if (!binaries.standalone.exists && !fs.existsSync(argv.seleniumServerJar)) {
```

Comment:
Shouldn't this be `!binaries.standalone.exists && (!argv.seleniumServerJar || !fs.existsSync(argv.seleniumServerJar))`?